packages/debs/SPECS/debian/rules

```diff
@@ -56,7 +57,11 @@ override_dh_install:
 	USER_AUTO_START="n" \
 	USER_CREATE_SSL_CERT="n" \
 	DOWNLOAD_CONTENT="y" \
-	./install.sh
+	./install.sh || { echo "install.sh failed! Aborting." >&2; exit 1; }
+
+	echo "Short commit hash: ${SHORT_COMMIT_HASH}"
+	sed -i '/"stage":/s/$$/,/; /"stage":/a \    "commit": "'"${SHORT_COMMIT_HASH}"'"' $(INSTALLATION_DIR)usr/share/wazuh-server/VERSION.json
+	cat $(INSTALLATION_DIR)usr/share/wazuh-server/VERSION.json
```

Comment:
Is this `cat` on purpose to show the file content during the package generation or should we remove it?